CesiumGltf/test/TestPropertyTypeTraits.cpp

```diff
@@ -193,365 +193,65 @@ TEST_CASE("Test MetadataArrayType") {
   REQUIRE(std::is_same_v<type, uint32_t>);
 }
 
-TEST_CASE("TypeToPropertyType") {
```

Comment:
Now that we're preserving `TypeToPropertyType`, we should preserve the unit tests for it too.